main.cpp

```diff
@@ -190,285 +205,320 @@ struct Safeway
+}
+
+void Artist::Group::changeNickname(std::string newNickname)
+{
+    std::cout << " New Nickname" << newNickname << std::endl;
+}
+
+std::string Artist::followAArtist(std::string nameOfArtist)
+{
+    return nameOfArtist;
+}
+
+void Artist::enterTheGroupOfAArtist(Group group)
+{
+    std::cout << " Successful enter into" << group.numberOfMembers << std::endl;
+}
+
+void Artist::CheckTheHottestSongOfArtist(int rankingOfASong)
+{
+    std::cout << " The ranking:" << rankingOfASong << std::endl;
+}
+
+struct Album//7
 {
-    //1)sales volume
     int salesVolume = 130023;
-    //2)release year
     int releaseYear = 1998;
-    //3)number of songs
     int numberOfSongs = 10;
-    //4)length of album
     float lengthOfAlbum = 32.5f;   
-    //5)price of album
     float priceOfAlbum = 34.99f;
-    //1)save the image of cover
     void saveTheImageOfCover(char addressOfCover);
-    //2)read the brief introduction of a album
     void ReadTheBriefIntroductionOfAAlbum(char briefIntroduction);
-    //3)play a song in album
     void playASongInAlbum (char addressOfSong);
 };
-/*
-8) Ranking 
-5 properties:
-    1)ranking of a song
-    2)number of ranking changes of a song
-    3)period of the ranking     
-    4)peak
-    5)the number of people who share the ranking to social platform
-3 things it can do:
-    1)play the music of a ranking
-    2)check a ranking of this week
-    3)Favorite a ranking
- */
-struct Ranking
+
+void Album::saveTheImageOfCover(char addressOfCover)
+{
+    
+    std::cout << addressOfCover << "succesfully saved" << std::endl;
+}
+
+void Album::ReadTheBriefIntroductionOfAAlbum(char briefIntroduction)
+{
+    std::cout << briefIntroduction << std::endl;
+}
+
+void Album::playASongInAlbum (char addressOfSong)
+{
+    std::cout << "visit" << addressOfSong << std::endl;
+}
+
+struct Ranking//8
 {
-    //1)ranking of a song
     int rankingOfASong = 17;
-    //2)number of ranking changes of a song
     int numberOfRankingChangesOfASong = -2;
-    //3)period of the ranking
     int periodOfTheRanking= 7;
-    //4)peak
     int peak = 3;
-    //5)number of people who share the ranking to social platform
+
     int numberOfPeoplaWhoShareTheRankingToSocialPlatform = 466;
-    //1)play the music of a ranking
-    void playTheMusicOfARanking(char nameOfASong);
-    //2)check a ranking of this week
+    bool playTheMusicOfARanking(std::string nameOfASong);
     void checkARankingOfThisWeek(int numberOfWeek);
-    //3)Favorite a ranking
-    void favoriteARanking(int numberOfPlaylist);
+    double favoriteARanking(int numberOfPlaylist);
 };
-/*
-9) Music interview
-5 properties:
-    1)host
-    2)artist    
-    3)number of word
-    4)date
-    5)topic
-3 things it can do:
-    1)Read the interview
-    2)comment on the interview
-    3)watch the video of the interview
- */
-struct MusicInterview
+
+bool Ranking::playTheMusicOfARanking(std::string nameOfASong)
+{
+    std::cout << nameOfASong << std::endl;
+    return 1;
+}
```

Comment:
use `true` or `false` when working with boolean values.
Don't use integers to represent booleans.
- [ ] revise